### PR TITLE
New Version of LINKAGES

### PR DESCRIPTION
**Pull request by araiho**
I changed the fortran code, so that it takes more flexible input files. I added the new executable to BETY. Then, I changed the write.configs and model2netcdf files to match the new version of fortran code.

### CODE REVIEW

File: models/ed/R/met2model.ED2.R [hunk 1]
```diff
@@ -267,7 +267,7 @@ for(year in start_year:end_year) {
 
   ## write DRIVER file
   sites <- 1
-  metgrid <- c(1,1,1,1,lon,lat)
+  metgrid <- c(1,1,1,1,floor(lon),floor(lat))
```
Comment (mdietze) [inline models/ed/R/met2model.ED2.R]: Ann, is there a reason for the change in how the ED met header file is written? @robkooper @bcow what did we determine was the correct setting here?

Comment (araiho) [inline models/ed/R/met2model.ED2.R]: I probably just accidentally changed it when I was referencing
met2model.ED2.R

Comment (robkooper) [inline models/ed/R/met2model.ED2.R]: I believe the lat/lon seemed to work for me best. The floor function would round it off and I would end up with case where the lat/lon would fall outside of the point area that ED uses. For example 0.9 would become 0, and ed would use -0.5 to 0.5 (or something along those lines).

Comment (mdietze) [inline models/ed/R/met2model.ED2.R]: Ann, if you could change back that one line and push it to Github I'd be happy to pull the rest. Thank's for the update!

Comment (araiho) [inline models/ed/R/met2model.ED2.R]: Yeah! I think I just pushed it. Should I submit another pull request?
